src/services/NetworkService.ts

```diff
@@ -168,4 +169,18 @@ export class NetworkService {
                   websocketInjected: WebSocket,
               });
     }
+    /**
+     * It checks if a node has Websocket functioning properly to subscribe.
+     * @param url the url.
+     */
+    public async checkWebsocketConnection(url: string) {
+        const webSocket = new WebSocket(url);
+        let websocketConnectionStatus: boolean = false;
+        webSocket.onmessage = function (e) {
+            // @ts-ignore
+            websocketConnectionStatus = e.currentTarget.readyState == 1;
+        };
+        await CommonHelpers.sleep(2000);
+        return { status: websocketConnectionStatus, sslNode: url.indexOf(':3001') !== -1 };
```

Comment:
Would be nice to extract `3000` and `3001` to constants and all references to them.